content/en/blog/2019/multicluster-version-routing/index.md

```diff
@@ -326,9 +326,9 @@ spec:
 EOF
 {{< /text >}}
 
-The address `224.0.0.3` of the service entry can be any arbitrary unallocated IP.
+The address `240.0.0.3` of the service entry can be any arbitrary unallocated IP.
 Note that loopback range `127.0.0.0/8` should not be used, as it will influence outbound traffic.
```

Comment:
Remove this sentence, because it's explained in the referenced multicluster doc?